dev/setup_env_osx.sh

```diff
@@ -1,15 +1,15 @@
 #!/bin/bash
-set -eo pipefail
+set -o errexit -o nounset -o pipefail
+
+readonly repo_root="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd -P)"
 
 command -v "python3" >/dev/null 2>&1 || {
-  echo >&2 "I require python3 but it's not installed. Officially supported version is 3.7.5"
+  echo >&2 "I require python3 but it's not installed. Officially supported version is $(cat "${repo_root}/../.python-version")"
```

Comment:
```suggestion
  echo >&2 "I require python3 but it's not installed. Officially supported version is $(cat "${repo_root}/.python-version")"
```

Fixing this in relationship to the corrected `repo_root` above in my suggestion, and to make it consistent with the other uses of `${repo_root}` below in this script.